### PR TITLE
Add Dockerfile with full features.

### PR DESCRIPTION
**Pull request by alvin1221**

### CODE REVIEW

**Review by JaylinYu — APPROVED** [review]
have to figure out how to enable FDB on docker

**Comment by Rory-Z** [comment]
What's the different for `deploy/docker/Dockerfile-full` and `deploy/docker/Dockerfile` ?

File: deploy/docker/Dockerfile-full [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+RUN cmake -G Ninja -DNOLOG=1 -DENABLE_JWT=ON \
+    -DNNG_ENABLE_TLS=ON \
+    -DNNG_ENABLE_SQLITE=ON \
+    -DBUILD_ZMQ_GATEWAY=ON \
+    -DBUILD_BENCH=ON ..
+
+RUN ninja install
+
+RUN ln -s /nanomq/build/nanomq/nanomq /usr/bin/nanomq
```
Comment (lee-emqx) [inline deploy/docker/Dockerfile-full:34]: nanomq can be installed to /usr/local/bin.

**Comment by Rory-Z** [comment]
> What's the different for `deploy/docker/Dockerfile-full` and `deploy/docker/Dockerfile` ?

Just check, looks `deploy/docker/Dockerfile-full` has `zeromq` more than `deploy/docker/Dockerfile` ?

**Comment by JaylinYu** [comment]
> 

? more than?

**Comment by alvin1221** [comment]
> > What's the different for `deploy/docker/Dockerfile-full` and `deploy/docker/Dockerfile` ?
> 
> Just check, looks `deploy/docker/Dockerfile-full` has `zeromq` more than `deploy/docker/Dockerfile` ?

Not only `zeromq`, but also complie `JWT`,  `TLS`, `SQLITE` and `Bench tool` by default.

**Comment by alvin1221** [comment]
> 

But there are a lot of issues on FDB compliation for now.

**Comment by alvin1221** [comment]
> have to figure out how to enable FDB on docker

But there are a lot of issues on FDB compliation for now.

**Pull request merged**